src/skin.cpp

```diff
@@ -49,6 +49,29 @@ Polygons SkinInfillAreaComputation::getInsidePolygons(const SliceLayerPart& part
  * This function is executed in a parallel region based on layer_nr.
  * When modifying make sure any changes does not introduce data races.
  *
+ * this function may only read/write the skin and infill from the *current* layer.
+ */
+Polygons SkinInfillAreaComputation::getOutlines(const SliceLayerPart& part_here, int layer2_nr)
```

Comment:
It wouldn't give a merge conflict if it gets removed here and added to the branch where it's necessary. Unless that branch then proceeds to modify the function, in which case you'd have to do a rebase to move the commit to the start of the branch.